api/pkg/apis/v1alpha1/utils/apiclient.go

```diff
@@ -93,6 +93,7 @@ type (
 		CreateCampaignContainer(ctx context.Context, instanceContainer string, payload []byte, namespace string, user string, password string) error
 		DeleteCampaignContainer(ctx context.Context, instanceContainer string, namespace string, user string, password string) error
 		GetCampaignContainer(ctx context.Context, instanceContainer string, namespace string, user string, password string) (model.CampaignContainerState, error)
+		CatalogOnConfig(ctx context.Context, name string, namespace string, user string, password string) (map[string]interface{}, error)
```

Comment:
Use a better name place. This name doesn't explain what the method and it doesn't follow the same naming pattern as other methods.